data-and-functions-1/src/functions/getActiveUsers.js

```diff
@@ -0,0 +1,13 @@
+import DATA from '../DATA'
```

Comment:
You don't need to pass in the data. The first parameter of all of your functions should be the data.